modules/intellij/updatePlugins_template.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is populated during publish stage and stored in a location that gets checked-in when publishing gh-pages.